classes/main.rb

```diff
@@ -0,0 +1,122 @@
+require './rental'
+require './person'
+require './teacher'
+require './student'
+require './book'
+
+def list_books(books)
+  puts 'List of books:'
+  books.each_with_index do |book, index|
+    puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}"
+  end
+end
+
+def list_people(people)
+  puts 'List of people:'
+  people.each_with_index do |person, index|
+    puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+  end
+end
+
+def create_book(books)
+  puts 'Please enter the title of the book:'
+  title = gets.chomp
+  puts 'Please enter the author of the book:'
+  author = gets.chomp
+  book = Book.new(title, author)
+  books.push(book)
+  puts 'Book created!'
+end
+
+def create_person(people)
+  puts 'Do you want to create a student(1) or a teacher(2)[Input the number]?'
+  type = gets.chomp
+  puts 'Please enter the name of the person:'
+  name = gets.chomp
+  puts 'Please enter the age of the person:'
+  age = gets.chomp
+  case type
+  when '1'
+    puts 'Please enter the classroom of the student:'
+    classroom = gets.chomp
+    person = Student.new(name, age, classroom)
+    people.push(person)
+    puts 'Student created!'
+  when '2'
+    puts 'Please enter the specializaation of the teacher:'
+    specializaation = gets.chomp
+    person = Teacher.new(name, age, specializaation)
+    people.push(person)
+    puts 'Teacher created!'
+  end
+end
+
+def create_rental(books, people)
+  puts 'Select a book from the following list by number(not Id):'
+  list_books(books)
+  book_index = gets.chomp.to_i
+  puts ''
+  puts 'Select a person from the following list by number(not Id):'
+  list_people(people)
+  person_index = gets.chomp.to_i
+  puts ''
+  puts 'Please enter the date of the rental:'
+  date = gets.chomp
+  puts ''
+  Rental.new(date, books[book_index], people[person_index])
+  puts 'Rental created!'
+end
+
+def list_rentals(people)
+  puts 'ID of the person:'
+  id = gets.chomp.to_i
+  person = people.find | person | person.id == id
```

Comment:
- [ ] This line of code is having problems, i think you were trying to use the `or` operator(`||`), instead you write `|` kindly fix this issue 😄
![imagen](https://user-images.githubusercontent.com/60789317/146585445-5dec6a51-6faf-4663-a40c-61b31c7670a9.png)